egs/bn_music_speech/v1/run.sh

```diff
@@ -0,0 +1,88 @@
+# Copyright 2015   David Snyder
+# Apache 2.0.
+#
+# This example demonstrates music/speech discrimination. This recipe trains
+# three GMMs on the music, speech and noise portions of the MUSAN corpus.
+# We test the systems on Broadcast News. The Broadcast News test data consists
+# of short segments of either speech or music. The classification decisions
+# are made at a segment level from the average likelihoods of two GMMs.
+# Results (EERs) are inline in comments below.
+#
+# See README.txt for more info on data required.
+
+. cmd.sh
+. path.sh
+set -e
+mfccdir=`pwd`/mfcc
+vaddir=`pwd`/mfcc
+
+local/make_bn.sh /export/corpora/NIST/96HUB4/h4eng_sp \
+  /export/corpora/LDC/LDC97T22/ data
+local/make_musan.sh /export/corpora/JHU/musan data
+
+steps/make_mfcc.sh --mfcc-config conf/mfcc.conf --nj 30 --cmd "$train_cmd" \
+    data/musan_speech exp/make_mfcc $mfccdir
+steps/make_mfcc.sh --mfcc-config conf/mfcc.conf --nj 30 --cmd "$train_cmd" \
+    data/musan_music exp/make_mfcc $mfccdir
+steps/make_mfcc.sh --mfcc-config conf/mfcc.conf --nj 5 --cmd "$train_cmd" \
+    data/musan_noise exp/make_mfcc $mfccdir
+steps/make_mfcc.sh --mfcc-config conf/mfcc.conf --nj 30 --cmd "$train_cmd" \
+    data/bn exp/make_mfcc $mfccdir
+
+utils/fix_data_dir.sh data/musan_speech
+utils/fix_data_dir.sh data/musan_music
+utils/fix_data_dir.sh data/musan_noise
+utils/fix_data_dir.sh data/bn
+
+sid/compute_vad_decision.sh --nj 20 --cmd "$train_cmd" \
+    data/musan_speech exp/make_vad $vaddir
+sid/compute_vad_decision.sh --nj 5 --cmd "$train_cmd" \
+    data/musan_noise exp/make_vad $vaddir
+sid/compute_vad_decision.sh --nj 20 --cmd "$train_cmd" \
+    data/musan_music exp/make_vad $vaddir
+sid/compute_vad_decision.sh --nj 20 --cmd "$train_cmd" \
+    data/bn exp/make_vad $vaddir
+
+sid/train_diag_ubm.sh --nj 10 --cmd "$train_cmd" --delta-window 2 \
+    data/musan_noise 32 exp/diag_ubm_noise &
+sid/train_diag_ubm.sh --nj 20 --cmd "$train_cmd" --delta-window 2 \
+    data/musan_speech 32 exp/diag_ubm_speech &
+sid/train_diag_ubm.sh --nj 20 --cmd "$train_cmd" --delta-window 2 \
+    data/musan_music 32  exp/diag_ubm_music
+wait;
+
+sid/train_full_ubm.sh --nj 20 --cmd "$train_cmd" \
+    --remove-low-count-gaussians false data/musan_noise \
+    exp/diag_ubm_noise exp/full_ubm_noise &
+sid/train_full_ubm.sh --nj 20 --cmd "$train_cmd" \
+    --remove-low-count-gaussians false data/musan_speech \
+    exp/diag_ubm_speech exp/full_ubm_speech &
+sid/train_full_ubm.sh --nj 20 --cmd "$train_cmd" \
+    --remove-low-count-gaussians false data/musan_music \
+    exp/diag_ubm_music exp/full_ubm_music
+wait;
+
+sid/music_id.sh --cmd "$train_cmd" --nj 40 \
+  exp/full_ubm_music exp/full_ubm_speech \
+  data/bn exp/bn_music_speech
+sid/music_id.sh --cmd "$train_cmd" --nj 40 \
+  exp/full_ubm_noise exp/full_ubm_speech \
+  data/bn exp/bn_noise_speech
+
+printf "EER using GMMs trained on music and speech"
+compute-eer <(python local/print_scores.py exp/bn_music_speech/ratio)
```

Comment:
we generally just invoke python scripts directly, relying on the shebang to identify it as a python script.